design/www/app/verification/verification.html

```diff
@@ -0,0 +1,31 @@
+<ion-view view-title="Login" id="background">
+  <ion-content scroll="false">
+     <div class="center-block w-xxl w-auto-xs p-v-md">
+    <div class="navbar">
+      <div class="navbar-brand text-center">
+        <span class="inline white">CHATT</span>
+      </div>
+    </div>
+
+    <div class="p-lg panel md-whiteframe-z1 text-color m">
+      <div class="m-b text-sm">
+        Verify your CHATT Account
+      </div>
+      <form name="form">
+        <div class="md-form-group float-label" ng-show="verification.showEmailInput">
+          <input type="text" class="md-input" ng-model="verification.data.email" required>
```

Comment:
No need to show input with email.
We already know email for previous step.
So it should be shown directly instead of input type